micro-rdk-installer/src/main.rs

```diff
@@ -214,7 +214,7 @@ fn write_credentials_to_app_binary(
     Ok(())
 }
 
-fn flash(args: WriteFlashArgs, config: &Config) -> Result<(), Error> {
+fn flash(args: WriteFlashArgs, config: &Config, app_path: PathBuf) -> Result<(), Error> {
```

Comment:
The bootloader/app_path is actually determined in L294 on the `match` and not written-back to the `args`, hence why I'm passing it through directly here.